lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -156,8 +156,6 @@ protected function doLoadMetadata($class, $parent, $rootEntityFound, array $nonS
                     $class->name,
                     end($nonSuperclassParents)
                 );
-                // enable this in 3.0
-                // throw MappingException::missingInheritanceTypeDeclaration(end($nonSuperclassParents), $class->name);
```

Comment:
`MappingException::missingInheritanceTypeDeclaration()` is also commented out, isn't it?